src/neptune/attributes/atoms/float.py

```diff
@@ -45,5 +50,16 @@ def assign(self, value: typing.Union[FloatVal, float, int], *, wait: bool = Fals
         if not isinstance(value, FloatVal):
             value = FloatVal(value)
 
+        if is_unsupported_float(value.value):
+            warn_once(
+                message=f"WARNING: The value you're trying to log is a nonstandard float value ({str(value.value)}) "
+                f"that is not currently supported. "
+                f"We'll add support for this type of value in the future. "
+                f"For now, you can use utils.stringify_unsupported() to log one or more values as strings: "
+                f"run['field'] = stringify_unsupported(float({str(value.value)})",
```

Comment:
Add missing closing parenthesis

```suggestion
                f"run['field'] = stringify_unsupported(float({str(value.value)}))",
```

Reply:
Added